broker-core/src/main/java/io/zeebe/broker/workflow/processor/CatchEventBehavior.java

```diff
@@ -128,6 +131,22 @@ public boolean occurEventForElement(
 
       return true;
 
+    } else if (elementInstance == null && tokenEvent == null && bpmnId != null) {
+      // if the event is a timer start event
+
+      final DeployedWorkflow latestWorkflow =
+          state.getWorkflowState().getLatestWorkflowVersionByProcessId(bpmnId);
+
+      if (latestWorkflow != null) {
+        workflowInstanceRecord.reset();
+        workflowInstanceRecord.setVersion(latestWorkflow.getVersion());
+        workflowInstanceRecord.setWorkflowKey(latestWorkflow.getKey());
+        workflowInstanceRecord.setBpmnProcessId(bpmnId);
+        workflowInstanceRecord.setPayload(eventPayload);
+        streamWriter.appendNewCommand(WorkflowInstanceIntent.CREATE, workflowInstanceRecord);
```

Comment:
Please note that a workflow can have multiple start events (i.e. multiple timer start events, or timer start events + none start event + message start events). How do you trigger the right start event?